src/devices/schneider_electric.ts

```diff
@@ -1861,6 +1861,36 @@ const definitions: DefinitionWithExtend[] = [
             commandsOnOff({endpointNames: ['switch_sw']}),
         ],
     },
+    {
+        zigbeeModel: ['E8331DST300ZB'],
+        model: 'E8331DST300ZB',
+        vendor: 'Schneider Electric',
+        description: 'Wiser AvatarOn 1G dimmer switch',
+        extend: [
+            deviceEndpoints({endpoints: {l1: 10}}),
```

Comment:
Is this really needed? Also remove ` endpointNames: ['l1'],` from `light({` below

Reply:
Have removed both deviceEndpoints and endpointNames and tested. Works. Thanks!